README.md

```diff
@@ -98,42 +88,341 @@ See [variables.tf] and [examples/] for details and use-cases.
+  Default is `{}`.
+
+##### Subnet Routing
+
+- **`route_table_tags`**: *(Optional `map(string)`)*
+
+  A map of tags to apply to all created Route Tables.
+  Default is `{}`.
+
+- **`public_route_table_tags`**: *(Optional `map(string)`)*
+
+  A map of tags to apply to the created Public Route Tables.
+  Default is `{}`.
+
+- **`private_route_table_tags`**: *(Optional `map(string)`)*
+
+  A map of tags to apply to the created Private Route Tables.
+  Default is `{}`.
+
+- **`intra_route_table_tags`**: *(Optional `map(string)`)*
+
+  A map of tags to apply to the created Intra Route Tables.
+  Default is `{}`.
 
 ## Module Attributes Reference
 
 The following attributes are exported by the module:
 
-- **`module_enabled`**
+- **`public_subnets_by_group`**
 
-  Whether this module is enabled.
+  A map of lists of public subnets keyed by group. (aws_subnet)
 
-- **`module_defaults`**
+- **`public_route_table_ids_by_group`**
 
-  Default settings that overwrite the module and resource defaults in this module.
+  A map of lists of public route table IDs keyed by group.
 
-- **`module_tags`**
+- **`public_subnet_ids_by_group`**
 
-  The map of tags that are being applied to all created resources that accept tags.
+  A map of lists of public subnet IDs keyed by group.
+
+- **`private_subnets_by_group`**
+
+  A map of lists of private subnets keyed by group. (aws_subnet)
+
+- **`private_route_table_ids_by_group`**
+
+  A map of lists of private route table IDs keyed by group.
+
+- **`private_subnet_ids_by_group`**
+
+  A map of lists of private subnet IDs keyed by group.
+
+- **`intra_subnets_by_group`**
+
+  A map of lists of intra aws_subnet keyed by group. (aws_subnet)
+
+- **`intra_route_table_ids_by_group`**
+
+  A map of lists of intra route table IDs keyed by group.
 
-- **`output_1`**
+- **`intra_subnet_ids_by_group`**
 
-  The full `resource` object with all its attributes.
+  A map of lists of intra subnet IDs keyed by group.
+
+- **`vpc`**
+
+  The VPC. (aws_vpc)
+
+- **`intra_route_tables`**
+
+  A map of intra route tables keyed by group. (aws_route_table)
+
+- **`intra_route_table_associations`**
+
+  A map of intra route table associations keyed by the subnets CIDR Blocks. (aws_route_table_association)
+
+- **`eips`**
+
+  A map of Elastic IP Adresses (EIPs) keyed by availability zone. (aws_eip)
+
+- **`nat_gateways`**
+
+  A map of NAT gatweways keyed by availability zone. (aws_nat_gateway)
+
+- **`private_route_tables`**
+
+  A map of private route tables keyed by group. (aws_route_table)
+
+- **`private_route_table_associations`**
+
+  A map of private route table associations keyed by the subnets CIDR Blocks. (aws_route_table_association)
+
+- **`routes_to_nat_gateways`**
+
+  A map of routes to the NAT Gateways keyed by group. (aws_route)
+
+- **`internet_gateway`**
+
+  The Internet Gateway. (aws_internet_gateway)
+
+- **`public_route_tables`**
+
+  A map of public route tables keyed by group. (aws_route_table)
```

Comment:
same as above

Reply:
this will be fun for you once you continue working on terradoc ;) hehe.. really wanna open this route? ;)

Reply:
not necessarily, plain text isn't enough though. If my suggestion isn't the right way of approaching things, we need to think about a different solution instead.